### PR TITLE
App Router: add site-wide metadata

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Migrate document meta tags to app router

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

File: apps/store/src/app/layout.tsx [hunk 1]
```diff
@@ -26,3 +26,29 @@ const Layout = ({ children }: PropsWithChildren) => {
...
+      sizes: '16x16',
+      url: '/favicon-16x16.png',
+    },
+    {
+      rel: 'mask-icon',
+      url: '/safari-pinned-tab.svg',
+      color: 'hsl(0, 0%, 7%)', // theme.colors.gray1000
+    },
+  ],
+  manifest: '/site.webmanifest',
```
Comment (alebedev) [inline apps/store/src/app/layout.tsx:53]: Is it automatically served?  Just curious

**Review by alebedev — APPROVED** [review]
Great!

File: apps/store/src/app/layout.tsx (hunk 1 above)
Comment (robinandeer) [reply apps/store/src/app/layout.tsx:53]: No we still need to put it in the public folder

**Review by guilhermespopolin — APPROVED** [review]

**Comment by robinandeer** [comment]
@robinandeer merged this pull request with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2876).

**Pull request merged**